csrankings.csv

```diff
@@ -10453,7 +10453,7 @@ Nathalie Rose T. Lim,De La Salle University,https://www.dlsu.edu.ph/colleges/ccs
 Nathaly Verwaal,University of Calgary,http://contacts.ucalgary.ca/info/cpsc/profiles/102-3419,NOSCHOLARPAGE
 Nathan Beckmann,Carnegie Mellon University,http://www.cs.cmu.edu/~beckmann,zkdjit0AAAAJ
 Nathan Brunelle,University of Virginia,https://engineering.virginia.edu/faculty/nathan-brunelle,NOSCHOLARPAGE
-Nathan D. Cahill,Rochester Institute of Technology,https://people.rit.edu/ndcsma,NOSCHOLARPAGE
+Nathan D. Cahill,Rochester Institute of Technology,https://people.rit.edu/ndcsma,https://scholar.google.com/citations?user=O7uS3IUAAAAJ&hl=en&oi=sra

```

Comment:
You need to only include the following text: O7uS3IUAAAAJ (and the corresponding other one).
Please fix and I will merge.

Reply:
Done